### ISSUE TITLE
[BUG] balance-tlb instead of active-backup in example config

### ISSUE THREAD

**Issue by Martin-Weiss** [harvester/harvester#6313]
**Describe the bug**
We had some issues with balance-tlb and due to that the default has been changed to active-backup some long time ago.
Unfortunately the docs have not been updated at https://docs.harvesterhci.io/v1.3/install/harvester-configuration and still show balance-tlb.

-> could someone search and replace in the docs from balance-tlb to active-backup?

**Expected behavior**
Default should be active-backup

**Comment by rrajendran17** [comment]
https://docs.harvesterhci.io/v1.4/install/harvester-configuration/
yes doc has to be corrected, moving to backlog

**Comment by Martin-Weiss** [comment]
Seems this is still wrong:

<img width="792" alt="image" src="https://github.com/user-attachments/assets/e78770ae-2aef-4618-a1c2-3f6cad9ec23f">

**Comment by rrajendran17** [comment]
Doc has to be be corrected.
https://docs.harvesterhci.io/v1.5/install/harvester-configuration/

**Comment by Martin-Weiss** [comment]
@rrajendran17  - just checked the docs - but there is still balance-tlb where it should be active-backup?!?

**Comment by rrajendran17** [comment]
> [@rrajendran17](https://github.com/rrajendran17) - just checked the docs - but there is still balance-tlb where it should be active-backup?!?

@Martin-Weiss yes the document needs correction.we will raise a doc PR for this.Thanks
